function-template-generator.js

```diff
@@ -38,11 +38,17 @@ function makePremble(funcDef) {
 
 
 function makeCall(funcDef) {
+	let paramString = "";
```

Comment:
When/if you get a chance, it'd be great to have more comments in here as to what these lines do, so that I or others can update them